diff --git a/third_party/boringssl-with-bazel/src/include/openssl/target.h b/third_party/boringssl-with-bazel/src/include/openssl/target-ppc64.h
index e5fcd99..ca430fb 100644
--- a/third_party/boringssl-with-bazel/src/include/openssl/target.h
+++ b/third_party/boringssl-with-bazel/src/include/openssl/target-ppc64.h
@@ -54,6 +54,8 @@
 #define OPENSSL_32_BIT
 #elif defined(__myriad2__)
 #define OPENSSL_32_BIT
+#elif defined(__powerpc64__)
+#define OPENSSL_64_BIT
 #else
 // The list above enumerates the platforms that BoringSSL supports. For these
 // platforms we keep a reasonable bar of not breaking them: automated test
